### PR TITLE
Validation if condition error improvements

### PR DESCRIPTION
**Pull request by gordonfarrell**
Improve error handling around validation failures and make it easier for users to find the error logs.

-Renamed the internal failure from the if statement from "Fail1" to "Message validation failure"
-Replaced the internal failure's unhelpful error message with the actual error output from the validation step of the pipeline.

### CODE REVIEW

File: terraform/modules/data_factory/ingestion-pipeline.json
Comment (DanPaseltiner) [inline terraform/modules/data_factory/ingestion-pipeline.json:1]: 🚀

**Review by DanPaseltiner — COMMENTED** [review]
This is definitely an improvement. Thanks for putting this together! Before merging could you do the following?

1. Deploy this branch to a dev environment.
2. Upload an invalid eICR.
3. Add a screen shot as comment on this PR demonstrating things are functioning as expected.

File: terraform/modules/data_factory/ingestion-pipeline.json
Comment (gordonfarrell) [reply terraform/modules/data_factory/ingestion-pipeline.json:1]: Screenshot 2023-05-31 at 9 11 06 AM
Works like a charm!

**Comment by gordonfarrell** [comment]
Whoops sticking the working screenshot here too since putting in the resolved convo hides it! 
Screenshot 2023-05-31 at 9 11 06 AM

**Review by DanPaseltiner — APPROVED** [review]
LGTM!

**Pull request merged**